packages/vscode-extension/package.json

```diff
@@ -830,6 +830,60 @@
         "scopeName": "teamsfx-toolkit.output",
         "path": "./syntaxes/teamsfx-toolkit-output.tmLanguage"
       }
+    ],
+    "walkthroughs": [
+      {
+        "id": "teamsToolkitQuickStart",
+        "title": "Get started with Teams toolkit  ",
+        "description": "Jumpstart your Teams app development experience   ",
+        "steps": [
+          {
+            "id": "teamsToolkitCapailities",
+            "title": "What are Teams app \"Capabilities\"",
+            "description": "[Capabilities](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/capabilities-overview) are the extension points for building apps on the Microsoft Teams platform.\n[Watch video (<1 min)](https://aka.ms/teamsfx-video)",
+            "media": {
+              "svg": "media/learn.svg",
+              "altText": "Watch"
+            }
+          },
+          {
+            "id": "teamsToolkitCommands",
+            "title": "Explore Teams toolkit commands",
+            "description": "[Open Command Palette (Ctrl+Shift+P)](command:workbench.action.quickOpen?%5B%22%3ETeams%22%5D) and type 'Teams' to find Teams toolkit commands. You can also discover [Teams toolkit icon](https://www.bing.com) on the activity bar to easily see the commands after a project creation. \n[Display Teams commands](command:workbench.action.quickOpen?%5B%22%3ETeams%22%5D)",
+            "media": {
+              "svg": "media/step_commands.svg",
+              "altText": "Display Teams Commands"
+            }
+          },
+          {
+            "id": "teamsToolkitInstallNodeJs",
+            "title": "Install Node.js",
+            "description": "Teams toolkit cannot detect the right version of Node.js on your machine.\nAs a fundamental runtime context for Teams app, Node.js v10.x, v12.x or v14.x is required (v.14.x is recommended ). Please install the appropriate version to run the Microsoft Teams Toolkit.Read more about managing Node.js versions.\n[Download Node.js (v.14.x)](https://nodejs.org/en/download/)",
+            "media": {
+              "svg": "media/step_nodejs.svg",
+              "altText": "Install NodeJs"
+            },
+            "when": "!fx-extension.isNotValidNode"
+          },
+          {
```

Comment:
since ITP are not going to ship out this sprint, I think we need to adjust the content here. @jaeyonglee8687 @yanjiang222